about/policy/open-sgid/index.md

```diff
@@ -17,11 +17,11 @@ The following guidelines apply when using the Open SGID database.
 
 - Search our [known-issues](https://github.com/agrc/open-sgid/issues) page if you are having trouble 
 - {% include contact.html subject=page.title text="Contact " contact=site.data.contacts.opensgid %} if you encounter a new issue.
-- Use for individual purposes only. Examples include:
-  - Cartography
-  - Geoprocessing
-  - Analysis
-  - Scripting
+- Use for a wide variety of individual purposes, such as:
+  - making cool maps
+  - solving problems
+  - writing nifty spatial queries
+  - exporing the power of open source
```

Comment:
see what you think @steveoh @nathankota

Reply:
...aside from the spelling error "exploring"

Reply:
**suggestion**: Maybe mix the professional terms in with the fancy ones so we don't sound too goofy. i'd probably nix nifty all together ... because ...  and then i would use the imperative `write` instead of `writing` with all the verbs.

Reply:
this ☝, and I like the rewording of the parent bullet

Reply:
good suggestions.  thx!